deploy/ft/ft-create-sdk.js

```diff
@@ -0,0 +1,49 @@
+//in this script, we check if the funder owns the NFTs they are adding to the contract. If they don't, throw an error
+const { FUNDING_ACCOUNT_ID, FUNDER_INFO, NETWORK_ID, NUM_KEYS, DROP_METADATA, DEPOSIT_PER_USE, DROP_CONFIG, KEYPOM_CONTRACT, NFT_DATA, NFT_CONTRACT_ID, NFT_METADATA} = require("./configurations");
+
+const path = require("path");
+const homedir = require("os").homedir();
+const { writeFile, mkdir, readFile } = require('fs/promises');
+const { initKeypom, createDrop, getDrops } = require("keypom-js");
+
+//funder is account to sign txns, defaults to benjiman.testnet
+//numKeys default is 10, deposit defalt is 10, default drop config 1 use per key and delete on empty, metadata empty, funderBalance false
+async function createNFTDropOwned({funderBalance = false}){
+    //USER'S RESPONSIBILITY TO CHANGE DEFAULT CONSTS IN CONFIGURATIONS.JS
+
+    //init keypom, this takes care of the new NEAR connection
+    console.log("Initiating NEAR connection");
+    initKeypom({network: NETWORK_ID, funder: FUNDER_INFO});
+
+	//get amount to transfer and see if owner has enough balance to fund drop
+	let amountToTransfer = new BN(FT_DATA.balancePerUse).mul(new BN(NUM_KEYS * DROP_CONFIG.usesPerKey))
+	let amountToTransfer_String = amountToTransfer.toString();
```

Comment:
This is redundant, just call the `.toString()` on line 19: `new BN(FT_DATA.balancePerUse).mul(new BN(NUM_KEYS * DROP_CONFIG.usesPerKey)).toString()` and keep the name as `amountToTransfer`, no need to have a separate variable explicitly saying it's a string.